docs/v1/api_bindings.md

```diff
@@ -47,6 +47,121 @@ curl -X POST -H "Content-Type: application/json"
  }
   ```
   
-  ### Errors
+### Errors
+Please refer to section [Errors](api_errors.md) to see all possible Errors
+  
+## [GET] Manage Bindings - ListAllBindings
+    
+ ### Request
+    
+ `GET /v1/bindings`
+    
+  ### Response
+     
+   If the request is successful, the response contains all the bindings.
+   
+   Success Response
+     
+   `200 OK`
+     
+```json
+  {
+      "bindings": [
+              {
+                  "name": "testb",
+                  "service_uuid": "uuid1",
+                  "host": "host1",
+                  "oidc_token": "testdn",
+                  "unique_key": "key",
+                  "created_on": "2018-05-23T09:25:25Z",
+                  "last_auth": "2018-05-23T09:25:25Z"
+              },
+              {
+                  "name": "testb2",
+                  "service_uuid": "uuid1",
+                  "host": "host1",
+                  "oidc_token": "testdn",
+                  "unique_key": "key",
+                  "created_on": "2018-05-23T09:25:43Z",
+                  "last_auth": "2018-05-23T09:25:25Z"
+              }
+      ]
+  }
+  ```
+  
+   ### Errors
+  
+   Please refer to section [Errors](api_errors.md) to see all possible Errors
+
+## [GET] Manage Bindings - ListAllBindingsByServiceTypeAndHost
+    
```

Comment:
add a description and update the title of the action to List all Bindings By Service Type And Host.